satpy/tests/enhancement_tests/test_enhancements.py

```diff
@@ -350,6 +350,31 @@ def test_cmap_vrgb_as_rgba(self):
             assert cmap.values[0] == 0
             assert cmap.values[-1] == 1.0
 
+    def test_cmap_with_alpha_set(self):
+        """Test that the min_alpha and max_alpha arguments set the alpha channel correctly."""
+        with closed_named_temp_file(suffix=".npy") as cmap_filename:
+            cmap_data = _generate_cmap_test_data(None, "RGB")
+            np.save(cmap_filename, cmap_data)
+            cmap = create_colormap({"filename": cmap_filename, "min_alpha": 100, "max_alpha": 255})
```

Comment:
Could the code below this block be de-dented for code clarity? The temp file is only needed when `create_colormap` is called right?

Reply:
Done, I've applied the same to the other similar neighbouring tests